docs/ReleaseNotes/Kubernetes-Operator-for-PostgreSQL-RN2.5.1.md

```diff
@@ -0,0 +1,29 @@
+# Percona Operator for PostgreSQL 2.5.1
+
+* **Date**
+
+    March 03, 2025
+
+* **Installation**
+
+    [Installing Percona Operator for PostgreSQL](../System-Requirements.md#installation-guidelines) 
+
+
+## Release highlights
+
+
+This release fixes the [CVE-2025-1094 :octicons-link-external-16:](https://www.postgresql.org/support/security/CVE-2025-1094/), vulnerability in the libpq PostgreSQL client library, which made images used by the Operator vulnerable to SQL injection within the PostgreSQL interactive terminal due to the lack of neutralizing quoting. For now, the fix includes the image of PostgreSQL 16.8  and other database cluster images based on PostgreSQL 16.8.
```

Comment:
i think we should also mention that we'll publish other versions in upcoming days